concent_api/api_testing_common.py

```diff
@@ -165,6 +165,8 @@ def _print_data(data, url):
     print()
     if response.status_code in [202, 204]:
         return None
```

Comment:
What about 200? Or 401? The whole 200..299 range should be treated as a success. 400..499 as a client error (so the test either has an error or is deliberately trying to cause a 4xx). 500..599 as a server-side bug that should be reported as an issue.

Reply:
resolved in https://github.com/golemfactory/concent/pull/773